### DISCUSSION TITLE
阿里云盘的令牌多久失效？还是服务重启后就失效？

### DISCUSSION THREAD

**Discussion by iemobiu** [AlistGo/alist#2557] [General]
如题。
阿里云盘的令牌多久失效呢？24小时还是48小时？还是服务重启后失效？
失效后可以重新获取，重新配置，但比较麻烦呀。这个令牌失效时长是多少呢？

**Comment by rakaki** [comment]
用了几个月了没失效

**Reply by iemobiu** [reply]
感谢，估计是我重启了好几次服务造成的。

**Reply by iemobiu** [reply]
另外问下，手机浏览器是不是无法播放视频呢？ MP4、mkv格式，iphone下用Safari和谷歌浏览器都是这样。电脑端浏览器也经常无法播放视频。是要怎么配置下吗？
